### PR TITLE
doc: Add swap example

### PR DESCRIPTION
**Pull request by howjmay**

### CODE REVIEW

File: sui-go/examples/swap/swap-go/main.go [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+	// 	utils.GetGitRoot()+"/examples/swap/swap",
+	// 	10*sui.DefaultGasBudget,
+	// 	&models.SuiTransactionBlockResponseOptions{
+	// 		ShowEffects:       true,
+	// 		ShowObjectChanges: true,
+	// 	},
+	// )
+	// if err != nil || !swapTxRes.Effects.Data.IsSuccess() {
+	// 	panic(err)
+	// }
```
Comment (dessaya) [inline sui-go/examples/swap/swap-go/main.go]: Will this code be un-commented soon? If not, can it be removed? Or add an explanation for why it is here.

File: sui-go/examples/swap/swap-go/main.go [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+	// 	context.Background(),
+	// 	signer.Address,
+	// 	txBytes,
+	// 	nil,
+	// 	nil,
+	// )
+	// if err != nil || !txnResponse.Effects.Data.IsSuccess() {
+	// 	fmt.Println("txnResponse.Effects.Data: ", txnResponse.Effects.Data.V1.Status.Error)
+	// 	panic(err)
+	// }
```
Comment (dessaya) [inline sui-go/examples/swap/swap-go/main.go]: Same here

**Review by dessaya — APPROVED** [review]
Looks good! Would be nice to add a small README or comment somewhere explaining the purpose of the `swap` example ("This example demonstrates this and that...") and how to compile and run.

File: sui-go/examples/swap/swap-go/main.go (hunk 1 above)
Comment (howjmay) [reply sui-go/examples/swap/swap-go/main.go]: oh it is wip. I want to add the swap_sui part, and maybe remove this part

**Pull request merged**